python/lsst/daf/butler/registries/remote.py

```diff
@@ -182,7 +218,12 @@ def refresh(self) -> None:
 
         # Need to determine what to refresh.
         # Might need a server method to return all the DatasetTypes up front.
-        pass
+        # How do we know which server should be refreshed?
+        # Should there be caches in the client?
+        response = self._client.put(self._get_url("v1/registry/refresh"))
```

Comment:
Yes, my comment in the code there was meant to be me worrying a lot about refresh but in this test code I really need refresh to run in the server because I inject the collections after the initial cached butler is created. You make a later comment on that.